### ISSUE TITLE
Support try/catch without error

### ISSUE THREAD

**Issue by OmgImAlexis** [ValYouW/njsTrace#16]
I had to change `} catch {` to `} catch(error) {` for this to start working again.

This is valid javascript, I'm not sure which specific spec adds support but I've been using this with node 12 for quite some time.

```js
try {
    // Handler non fatal errors
    $injector.resolve('globalErrorHandler')(coreError);
} catch {
    throw coreError;
}
```

```js
> require('./app')
njsTrace: Instrumenting: /usr/local/node/graphql-api/app/index.js
njsTrace: ERROR: Error instrumenting file: /usr/local/node/graphql-api/app/index.js , Exception: { SyntaxError: Unexpected token (75:10)
    at Parser.pp$4.raise (/usr/local/node/graphql-api/node_modules/falafel/node_modules/acorn/dist/acorn.js:2757:13)
    at Parser.pp.unexpected (/usr/local/node/graphql-api/node_modules/falafel/node_modules/acorn/dist/acorn.js:647:8)
    at Parser.pp$1.parseTryStatement (/usr/local/node/graphql-api/node_modules/falafel/node_modules/acorn/dist/acorn.js:1018:49)
    at Parser.pp$1.parseStatement (/usr/local/node/graphql-api/node_modules/falafel/node_modules/acorn/dist/acorn.js:784:32)
    at Parser.pp$1.parseBlock (/usr/local/node/graphql-api/node_modules/falafel/node_modules/acorn/dist/acorn.js:1112:23)
    at Parser.pp$3.parseFunctionBody (/usr/local/node/graphql-api/node_modules/falafel/node_modules/acorn/dist/acorn.js:2600:22)
    at Parser.pp$3.parseArrowExpression (/usr/local/node/graphql-api/node_modules/falafel/node_modules/acorn/dist/acorn.js:2561:8)
    at Parser.pp$3.parseExprAtom (/usr/local/node/graphql-api/node_modules/falafel/node_modules/acorn/dist/acorn.js:2136:23)
    at Parser.pp$3.parseExprSubscripts (/usr/local/node/graphql-api/node_modules/falafel/node_modules/acorn/dist/acorn.js:2047:19)
    at Parser.pp$3.parseMaybeUnary (/usr/local/node/graphql-api/node_modules/falafel/node_modules/acorn/dist/acorn.js:2024:17)
  pos: 1935,
  loc: Position { line: 75, column: 10 },
  raisedAt: 1936 }
{}
```

**Comment by ValYouW** [comment]
Good catch :)
I'll take a look when free.

**Comment by ValYouW** [comment]
Fixed in v1.2.0
I had to delete my package-lock file and re-install modules to get the latest 5.x version of the acorn module (this is not a direct dependency of njstrace but of falafel, and since falafel requires just acorn ^5.0.0 it might be possible that npm will not install the latest 5.x if there is already acorn installed, so just delete node_modules and package-lock. this is not the best solution but it's quick...)
